src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -623,22 +655,35 @@ public void BuildCFIMap(NodeFactory factory, ObjectNode node)
             EnsureCurrentSection();
         }
 
-        public void EmitCFICodes(int offset)
+        public void EmitCFICodes(int offset, NodeFactory factory)
```

Comment:
Target platform is available in the `_targetPlatform` instance variable. This parameter seems unnecessary.